Failed to install wpilib

Package: wpilib:arm64-osx -> main

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: dafef74af 2023-07-07 (2 days ago)

**To Reproduce**

`vcpkg x-set-installed wpilib`

**Failure logs**

```
-- Downloading https://github.com/wpilibsuite/allwpilib/archive/d37f990ce3a8a36bc791dd989dcea2868759e556.tar.gz -> wpilibsuite-allwpilib-d37f990ce3a8a36bc791dd989dcea2868759e556.tar.gz...
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/wpilibsuite-allwpilib-d37f990ce3a8a36bc791dd989dcea2868759e556.tar.gz
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean
-- Configuring arm64-osx
CMake Warning at vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:318 (message):
  The following variables are not used in CMakeLists.txt:

      FLAT_INSTALL_WPILIB

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/wpilib/portfile.cmake:18 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)


-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/install-arm64-osx-dbg-out.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/install-arm64-osx-dbg-err.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/wpilib/portfile.cmake:29 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/install-arm64-osx-dbg-out.log</summary>

```
[1/267] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/arm64-osx-dbg/wpiutil && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -Dinput=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/resources/wpilib-128.png -Doutput=generated/main/cpp/wpilib-128.png.cpp -Dprefix=WPI -Dnamespace=wpi -P /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/cmake/modules/../scripts/GenResource.cmake
[2/267] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/fmtlib/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/memory/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/json/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/llvm/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/mpack/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/sigslot/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -pedantic -Wextra -Werror -Wno-unused-parameter -Wno-deprecated-anon-enum-enum-conversion -std=gnu++20 -MD -MT wpiutil/CMakeFiles/wpiutil.dir/src/main/native/cpp/timestamp.cpp.o -MF wpiutil/CMakeFiles/wpiutil.dir/src/main/native/cpp/timestamp.cpp.o.d -o wpiutil/CMakeFiles/wpiutil.dir/src/main/native/cpp/timestamp.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/cpp/timestamp.cpp
[3/267] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/fmtlib/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/memory/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/json/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/llvm/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/mpack/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/thirdparty/sigslot/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -pedantic -Wextra -Werror -Wno-unused-parameter -Wno-deprecated-anon-enum-enum-conversion -std=gnu++20 -MD -MT wpiutil/CMakeFiles/wpiutil.dir/src/main/native/cpp/MappedFileRegion.cpp.o -MF wpiutil/CMakeFiles/wpiutil.dir/src/main/native/cpp/MappedFileRegion.cpp.o.d -o wpiutil/CMakeFiles/wpiutil.dir/src/main/native/cpp/MappedFileRegion.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpiutil/src/main/native/cpp/MappedFileRegion.cpp
...
Skipped 536 lines
...

-- Up-to-date: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/BooleanArrayTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/DoubleTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/RawTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/RawTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/BooleanTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/IntegerTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/FloatArrayTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/FloatTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/DoubleArrayTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/IntegerArrayTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/BooleanArrayTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/StringArrayTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/DoubleArrayTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/FloatArrayTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/StringTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/IntegerTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/StringArrayTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/DoubleTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/BooleanTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/StringTopic.inc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/IntegerArrayTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/networktables/FloatTopic.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/ntcore_c_types.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/ntcore/ntcore_cpp_types.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/share/ntcore/ntcore-config.cmake
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/share/ntcore/ntcore.cmake
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/share/ntcore/ntcore-debug.cmake
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/MatrixFunctions
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/src
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/src/MatrixFunctions
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/src/MatrixFunctions/StemFunction.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/src/MatrixFunctions/MatrixLogarithm.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/src/MatrixFunctions/MatrixFunction.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/src/MatrixFunctions/MatrixPower.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/src/MatrixFunctions/MatrixExponential.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/unsupported/Eigen/src/MatrixFunctions/MatrixSquareRoot.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/Core
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/SparseLU
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/OrderingMethods
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/Householder
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/SparseCore
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/Jacobi
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/QR
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/SparseQR
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/SVD
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/Cholesky
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/IterativeLinearSolvers
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/LU
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/SparseCholesky
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/Eigenvalues
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/misc
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/misc/Kernel.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/misc/RealSvd2x2.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/misc/Image.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Reverse.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/ConditionEstimator.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/PermutationMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Assign.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/MathFunctionsImpl.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/CwiseUnaryOp.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/DiagonalMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/StlIterators.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/VectorBlock.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/ReturnByValue.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Solve.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/SelfadjointRank2Update.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/GeneralBlockPanelKernel.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/GeneralMatrixMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/Parallelizer.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/GeneralMatrixMatrixTriangular.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/TriangularMatrixVector.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/SelfadjointMatrixVector.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/TriangularSolverMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/GeneralMatrixVector.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/SelfadjointProduct.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/TriangularSolverVector.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/TriangularMatrixMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/products/SelfadjointMatrixMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/ArithmeticSequence.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Product.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/ReshapedHelper.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/BlasUtil.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/StaticAssert.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/Constants.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/XprHelper.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/Macros.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/DisableStupidWarnings.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/SymbolicIndex.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/ForwardDeclarations.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/Memory.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/Meta.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/ConfigureVectorization.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/IntegralConstant.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/ReenableStupidWarnings.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/util/IndexedViewHelper.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/MatrixBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Stride.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Map.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/ForceAlignedAccess.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/MathFunctions.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/functors
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/functors/UnaryFunctors.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/functors/TernaryFunctors.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/functors/AssignmentFunctors.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/functors/BinaryFunctors.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/functors/StlFunctors.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/functors/NullaryFunctors.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/SelfCwiseBinaryOp.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/BandMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Inverse.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Swap.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/ProductEvaluators.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/MapBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Matrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/EigenBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/StableNorm.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/GenericPacketMath.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/CwiseBinaryOp.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Visitor.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/DenseStorage.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/DiagonalProduct.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/CwiseUnaryView.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/NoAlias.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/DenseCoeffsBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/PartialReduxEvaluator.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/IndexedView.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Fuzzy.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/TriangularMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/IO.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/SSE
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/SSE/PacketMath.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/SSE/MathFunctions.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/SSE/TypeCasting.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/SSE/Complex.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/Default
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/Default/Half.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/Default/Settings.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/Default/TypeCasting.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/Default/BFloat16.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/Default/GenericPacketMathFunctionsFwd.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/Default/GenericPacketMathFunctions.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/Default/ConjHelper.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/AVX
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/AVX/PacketMath.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/AVX/MathFunctions.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/AVX/TypeCasting.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/AVX/Complex.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/NEON
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/NEON/GeneralBlockPanelKernel.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/NEON/PacketMath.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/NEON/MathFunctions.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/NEON/TypeCasting.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/arch/NEON/Complex.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Array.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/SolverBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/AssignEvaluator.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Replicate.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/CoreIterators.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/ArrayBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Block.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Dot.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/PlainObjectBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/SelfAdjointView.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/GeneralProduct.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/SolveTriangular.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/CoreEvaluators.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/CwiseNullaryOp.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/VectorwiseOp.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/ArrayWrapper.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/BooleanRedux.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Transpositions.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/NestByValue.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Ref.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Reshaped.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Transpose.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Redux.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/NumTraits.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/CommaInitializer.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/GlobalFunctions.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Random.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Diagonal.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/DenseBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/CwiseTernaryOp.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Core/Select.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_Utils.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_kernel_bmod.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_column_bmod.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_gemm_kernel.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_column_dfs.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_panel_dfs.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_relax_snode.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_heap_relax_snode.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_panel_bmod.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_Structs.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_pruneL.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_copy_to_ucol.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_Memory.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_pivotL.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLUImpl.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseLU/SparseLU_SupernodalMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/OrderingMethods
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/OrderingMethods/Ordering.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/OrderingMethods/Eigen_Colamd.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/OrderingMethods/Amd.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/ArrayCwiseUnaryOps.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/CommonCwiseUnaryOps.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/BlockMethods.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/ArrayCwiseBinaryOps.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/MatrixCwiseBinaryOps.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/IndexedViewMethods.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/CommonCwiseBinaryOps.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/ReshapedMethods.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/plugins/MatrixCwiseUnaryOps.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Householder
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Householder/HouseholderSequence.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Householder/Householder.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Householder/BlockHouseholder.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseSelfAdjointView.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/CompressedStorage.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseCwiseBinaryOp.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseView.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseColEtree.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseMatrixBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseDiagonalProduct.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseTriangularView.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseAssign.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparsePermutation.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseCwiseUnaryOp.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseMap.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseDenseProduct.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseRef.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseFuzzy.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/MappedSparseMatrix.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseSparseProductWithPruning.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseSolverBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseBlock.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/TriangularSolver.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseVector.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseCompressedBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseTranspose.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseProduct.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/AmbiVector.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseRedux.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/ConservativeSparseSparseProduct.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseUtil.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCore/SparseDot.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Jacobi
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Jacobi/Jacobi.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/QR
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/QR/FullPivHouseholderQR.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/QR/HouseholderQR.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/QR/CompleteOrthogonalDecomposition.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/QR/ColPivHouseholderQR.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseQR
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseQR/SparseQR.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SVD
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SVD/SVDBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SVD/JacobiSVD.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SVD/BDCSVD.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SVD/UpperBidiagonalization.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Cholesky
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Cholesky/LDLT.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Cholesky/LLT.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers/BasicPreconditioners.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers/IncompleteCholesky.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers/SolveWithGuess.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers/IncompleteLUT.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers/LeastSquareConjugateGradient.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers/ConjugateGradient.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers/BiCGSTAB.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/IterativeLinearSolvers/IterativeSolverBase.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/LU
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/LU/InverseImpl.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/LU/Determinant.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/LU/arch
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/LU/arch/InverseSize4.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/LU/PartialPivLU.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/LU/FullPivLU.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCholesky
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCholesky/SimplicialCholesky_impl.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/SparseCholesky/SimplicialCholesky.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/GeneralizedEigenSolver.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/EigenSolver.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/RealSchur.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/Tridiagonalization.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/HessenbergDecomposition.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/ComplexEigenSolver.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/GeneralizedSelfAdjointEigenSolver.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/ComplexSchur.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/SelfAdjointEigenSolver.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/RealQZ.h
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/packages/wpilib_arm64-osx/debug/include/wpimath/Eigen/src/Eigenvalues/MatrixBaseEigenvalues.h
FAILED: CMakeFiles/install.util 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/arm64-osx-dbg && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
```
</details>
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/install-arm64-osx-dbg-err.log</summary>

```
CMake Error at wpimath/cmake_install.cmake:45 (file):
  file INSTALL cannot find
  "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpimath/src/main/native/thirdparty/drake/include":
  No such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:49 (include)
```
</details>

**Additional context**

But the error
```
CMake Error at wpimath/cmake_install.cmake:45 (file):
  file INSTALL cannot find
  "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/wpilib/src/868759e556-81ce2d8562.clean/wpimath/src/main/native/thirdparty/drake/include":
  No such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:49 (include)
```
should not be vcpkg specific

--- a/ThirdPartyNotices.txt
+++ b/ThirdPartyNotices.txt
@@ -1066,41 +1066,6 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
-=============
-Drake Library
-=============
-All components of Drake are licensed under the BSD 3-Clause License
-shown below. Where noted in the source code, some portions may
-be subject to other permissive, non-viral licenses.
-
-Copyright 2012-2016 Robot Locomotion Group @ CSAIL
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are
-met:
-
-Redistributions of source code must retain the above copyright notice,
-this list of conditions and the following disclaimer.  Redistributions
-in binary form must reproduce the above copyright notice, this list of
-conditions and the following disclaimer in the documentation and/or
-other materials provided with the distribution.  Neither the name of
-the Massachusetts Institute of Technology nor the names of its
-contributors may be used to endorse or promote products derived from
-this software without specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
-"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
-LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
-A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
-HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
-SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
 =====================
 Portable File Dialogs
 =====================

--- a/ThirdPartyNotices.txt
+++ b/ThirdPartyNotices.txt
@@ -41,9 +41,6 @@ Team 254 Library      wpilibj/src/main/java/edu/wpi/first/wpilibj/spline/SplineP
                       wpilibc/src/main/native/include/trajectory/TrajectoryParameterizer.h
                       wpilibc/src/main/native/cpp/trajectory/TrajectoryParameterizer.cpp
 Portable File Dialogs wpigui/src/main/native/include/portable-file-dialogs.h
-Drake                 wpimath/src/main/native/thirdparty/drake/
-                      wpimath/src/test/native/cpp/drake/
-                      wpimath/src/test/native/include/drake/
 V8 export-template    wpiutil/src/main/native/include/wpi/SymbolExports.h
 GCEM                  wpimath/src/main/native/thirdparty/gcem/include/
 

--- a/docs/build.gradle
+++ b/docs/build.gradle
@@ -67,9 +67,6 @@ doxygen {
     cppIncludeRoots << '../ntcore/build/generated/main/native/include/'
 
     if (project.hasProperty('docWarningsAsErrors')) {
-        // Drake
-        exclude 'drake/common/**'
-
         // Eigen
         exclude 'Eigen/**'
         exclude 'unsupported/**'

--- a/wpimath/CMakeLists.txt
+++ b/wpimath/CMakeLists.txt
@@ -89,8 +89,7 @@ if (WITH_JAVA)
 
 endif()
 
-file(GLOB_RECURSE wpimath_native_src src/main/native/cpp/*.cpp
-                                     src/main/native/thirdparty/drake/src/*.cpp)
+file(GLOB_RECURSE wpimath_native_src src/main/native/cpp/*.cpp)
 list(REMOVE_ITEM wpimath_native_src ${wpimath_jni_src})
 
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS FALSE)
@@ -117,10 +116,6 @@ else()
     find_package(Eigen3 CONFIG REQUIRED)
     target_link_libraries (wpimath Eigen3::Eigen)
 endif()
-
-install(DIRECTORY src/main/native/thirdparty/drake/include/ DESTINATION "${include_dest}/wpimath")
-target_include_directories(wpimath SYSTEM PUBLIC
-                           $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/main/native/thirdparty/drake/include>)
 
 install(DIRECTORY src/main/native/thirdparty/gcem/include/ DESTINATION "${include_dest}/wpimath")
 target_include_directories(wpimath SYSTEM PUBLIC

--- a/wpimath/build.gradle
+++ b/wpimath/build.gradle
@@ -9,30 +9,11 @@ ext {
 
     nativeName = 'wpimath'
     devMain = 'edu.wpi.first.math.DevMain'
-
-    splitSetup = {
-        it.sources {
-            drakeCpp(CppSourceSet) {
-                source {
-                    srcDirs 'src/main/native/thirdparty/drake/src'
-                    include '**/*.cpp'
-                }
-                exportedHeaders {
-                    srcDirs 'src/main/native/thirdparty/drake/include',
-                            'src/main/native/thirdparty/eigen/include',
-                            'src/main/native/thirdparty/gcem/include'
-                }
-            }
-        }
-    }
 }
 
 apply from: "${rootDir}/shared/jni/setupBuild.gradle"
 
 cppHeadersZip {
-    from('src/main/native/thirdparty/drake/include') {
-        into '/'
-    }
     from('src/main/native/thirdparty/eigen/include') {
         into '/'
     }
@@ -47,7 +28,6 @@ model {
             it.sources.each {
                 it.exportedHeaders {
                     srcDirs 'src/main/native/include',
-                            'src/main/native/thirdparty/drake/include',
                             'src/main/native/thirdparty/eigen/include',
                             'src/main/native/thirdparty/gcem/include'
                 }

--- a/wpimath/src/test/native/cpp/estimator/MerweScaledSigmaPointsTest.cpp
+++ b/wpimath/src/test/native/cpp/estimator/MerweScaledSigmaPointsTest.cpp
@@ -6,8 +6,6 @@
 
 #include "frc/estimator/MerweScaledSigmaPoints.h"
 
-namespace drake::math {
-namespace {
 TEST(MerweScaledSigmaPointsTest, ZeroMean) {
   frc::MerweScaledSigmaPoints<2> sigmaPoints;
   auto points = sigmaPoints.SquareRootSigmaPoints(
@@ -30,5 +28,3 @@ TEST(MerweScaledSigmaPointsTest, NonzeroMean) {
                                    {2.0, 2.0, 2.00548, 2.0, 1.99452}})
           .norm() < 1e-3);
 }
-}  // namespace
-}  // namespace drake::math

